js/utils/timestamp.js

```diff
@@ -7,6 +7,12 @@ o2.Utilities.phpToMoment = function( s ) {
 	var m = '';
 	var lookBehind = '';
 	for ( var i = 0; i < s.length; i++ ) {
+		if ( lookBehind === "\\" ) { // Scaping character
+			m += '[' + s.charAt( i ) + ']';
+			lookBehind = s.charAt( i );
```

Comment:
Do we want to do this? There are checks for sequences like `jS`, but if the `j` has been escaped then it may not be appropriate to trigger the substitution. Perhaps `lookBehind` should be empty?